ui/index.less

```diff
@@ -33,6 +33,7 @@ tbody {
   font-weight: 300 !important;
   padding: 8px 0 4px 0 !important;
   margin: 0 32px !important;
+  cursor: default !important;
```

Comment:
@shafi1379 this is set at the body level, so does this affect any other hovers?

Reply:
@shafi1379 , I agree with @afoerster 's opinion. I think it maybe affect any other hovers.

Reply:
@afoerster @rk-fsdev Added specifically for `.ant-tabs-tab` class not for tbody. Please expand the file and see.

Reply:
![ARCH-687](https://user-images.githubusercontent.com/11935965/75949853-802d9900-5ecd-11ea-97fc-9c26d650348a.gif)